cpp/src/IceGrid/AdminSessionI.h

```diff
@@ -26,7 +26,7 @@ namespace IceGrid
 
         Ice::ObjectPrx _register(const std::shared_ptr<SessionServantManager>&, const Ice::ConnectionPtr&);
 
-        void keepAlive(const Ice::Current& current) override { BaseSessionI::keepAlive(current); }
+        void keepAlive(const Ice::Current&) override {} // no-op
```

Comment:
Maybe we should expand the comment to say why. Might not be obvious looking back at this in the future.

Reply:
Fixed comment.